includes/admin/customers/customers.php

```diff
@@ -344,16 +344,19 @@ function edd_customers_view( $customer = null ) {
 	$agreement_timestamps = $customer->get_meta( 'agree_to_terms_time',   false );
 	$privacy_timestamps   = $customer->get_meta( 'agree_to_privacy_time', false );
 
-	$last_payment = edd_get_payments( array(
-		'output'   => 'payments',
-		'post__in' => $customer->get_payment_ids(),
-		'orderby'  => 'date',
-		'number'   => 1
-	) );
+	$last_payment = edd_get_orders(
+		array(
+			'customer_id' => $customer->id,
+			'id__in'      => $customer->get_payment_ids(),
```

Comment:
This shouldn't be necessary because it's already covered by the `customer_id` argument.